go-controller/pkg/config/config.go

```diff
@@ -206,7 +206,7 @@ type KubernetesConfig struct {
 
 // OVNKubernetesFeatureConfig holds OVN-Kubernetes feature enhancement config file parameters and command-line overrides
 type OVNKubernetesFeatureConfig struct {
-	EgressIPEnabled bool `gcfg:"egress-ip-enable"`
+	EgressIPEnabled bool `gcfg:"enable-egress-ip"`
```

Comment:
should you rename the struct field too?

Reply:
yeah, i thought about it.. but just changed the user facing elements.. let me also change the struct field

Reply:
@danwinship PTAL